tasks/agent6-win.yml

```diff
@@ -0,0 +1,52 @@
+---
+- name: Create main Datadog agent configuration file
```

Comment:
Do we care about the owner of those files ? Aren't they going to belong to `administrator` ?

Reply:
They should inherit the rights from the directory, which the installer creates.

Reply:
Good to know !